src/docs.js

```diff
@@ -165,7 +166,8 @@ async function downloader(options) {
 
   if (options.testing) {
     let file = path.join(__dirname, '..', 'test', 'test_tarball.tar.gz');
-    readStream = fs.createReadStream(file);
+    readStream = awsMock.mock('S3', 'getObject', fs.createReadStream(file));
```

Comment:
I think the issue here is that `S3.getObject` is supposed to return an object with a `createReadStream` property that is a function returning a stream.  But your mock is having `S3.getObject` return a stream.

Reply:
humm... ok will dig more into the getObject function :)